### PR TITLE
Add Alibaba Cloud meeting info

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
Add Alibaba Cloud meeting info

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -717,6 +717,15 @@ sigs:
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/alibaba-cloud-csi-driver/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes/cloud-provider-alibaba-cloud/master/OWNERS
+    meetings:
+    - description: Regular Alibaba Cloud Subproject Meeting
+      day: Tuesday
+      time: "4:00"
+      tz: PT (Pacific Time)
```
Comment (cheftako) [inline sigs.yaml]: 4 am makes me suspect that pacific time is not the right time zone for the intended audience. I would suggest either using UTC or the time zone of the intended audience.

Comment (zhangxiaoyu-zidif) [reply sigs.yaml]: OK, we will use the time zone of the intended audience.

Comment (zhangxiaoyu-zidif) [reply sigs.yaml]: thetimezoneconverter seems not to recognize CST(China Standard Time), so I use UTC instead.

Comment (idealhack) [reply sigs.yaml]: you can use [`China`](http://www.thetimezoneconverter.com/?t=20:00&tz=China) instead of `CST`.

File: sigs.yaml [hunk 2]
```diff
@@ -717,6 +717,15 @@ sigs:
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/alibaba-cloud-csi-driver/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes/cloud-provider-alibaba-cloud/master/OWNERS
+    meetings:
+    - description: Regular Alibaba Cloud Subproject Meeting
+      day: Tuesday
+      time: "12:00"
+      tz: UTC
+      frequency: 'monthly 2020 start date: Jan. 7th'
+      url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
```
Comment (idealhack) [inline sigs.yaml]: I see this link also used by the other SIGs, maybe you can create your own?

File: sigs.yaml (hunk 1 above)
Comment (zhangxiaoyu-zidif) [reply sigs.yaml]: @idealhack 
Hello, it showed：
image

File: sigs.yaml (hunk 2 above)
Comment (zhangxiaoyu-zidif) [reply sigs.yaml]: This is not edited by me. I just edit sigs.yaml. This line was added automatically by `make`.

Comment (zhangxiaoyu-zidif) [reply sigs.yaml]: This link was used by other cloud providers, I think maybe this is common tradition. Is there any recommended url？

**Comment by zhangxiaoyu-zidif** [comment]
ping @idvoretskyi  please help to merge this PR ：）

File: sigs.yaml (hunk 2 above)
Comment (cheftako) [reply sigs.yaml]: The contents of that doc suggest that you should get a WG level doc soon. I think its fine for now but we should probably fix this once you get that doc.

File: sigs.yaml (hunk 1 above)
Comment (cheftako) [reply sigs.yaml]: Hotel or Hong Kong would probably work if they are acceptable. UTC is obviously fine.

**Comment by zhangxiaoyu-zidif** [comment]
ping @nikhita for approve :)

**Pull request merged**